extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -175,7 +175,7 @@ class AdSenseNetworkConfig {
    */
   isResponsiveEnabled(win) {
     const branch = getAdSenseAmpAutoAdsResponsiveExperimentBranch(win);
-    return branch != AdSenseAmpAutoAdsResponsiveBranches.CONTROL;
+    return branch != AdSenseAmpAutoAdsResponsiveBranches.EXPERIMENT;
```

Comment:
Perhaps I'm reading this wrong but looks to me like previously isResponsiveEnabled would have returned true for all of the traffic that was not in either the experiment or control, no?

Reply:
Yes that's true. I had followed the example of another experiment when I wrote this and I thought there is always a branch picked. Is that not the case? Because in that case I have to change the new logic as well.

Reply:
Given you want to launch this, it's fine.  Meant it more as a cautionary tale as it means you actually released your change to the vast majority of traffic when you thought it was behind an experiment.